functions/src/utils.ts

```diff
@@ -6,12 +6,10 @@ import {
 } from "firebase-functions/https";
 import { ZodError } from "zod";
 
-type Response = Record<string, unknown>;
-
 type Handler = (
 	req: CallableRequest<unknown>,
 	res: CallableResponse<unknown> | undefined,
-) => Promise<Response> | Response;
+) => void;
```

Comment:
Some of the functions [return a boolean](https://github.com/KonferCA/HawkHacks-Dashboard/blob/8aa981fdb01c9029ddd201f7442eeccb8213023b/functions/src/index.ts#L329) [or nothing](https://github.com/KonferCA/HawkHacks-Dashboard/blob/8aa981fdb01c9029ddd201f7442eeccb8213023b/functions/src/index.ts#L337-L358) which conflicted with this method signature which expected an object to be returned.

I think it's fine to let the handlers return anything, so I changed this to void